ci-operator/step-registry/aro/provision/nsg/aro-provision-nsg-commands.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+#echo "Installing oc binary"
+#curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz | tar zxvf - oc
+#chmod +x oc
+CLUSTER=${CLUSTER:="${NAMESPACE}-${UNIQUE_HASH}"}
```

Comment:
The step does not declare `CLUSTER` env, so I think it should be:
```suggestion
CLUSTER="${NAMESPACE}-${UNIQUE_HASH}"
```